src/Microsoft.AspNetCore.Hosting/project.json

```diff
@@ -50,7 +52,7 @@
         }
       }
     },
-    "netstandard1.3": {
+    "netstandard1.5": {
```

Comment:
:crying_cat_face: 

Can you double compile for `netstandard1.3` and `netstandard1.5`; saying `netstandard1.5` will give better diagnostics and `#if` `Assembly.Location`?

Or would that not work?